src/components/RedLiner/RedLiner.tsx

```diff
@@ -1,21 +1,16 @@
 import _ from 'lodash';
 import React, { useEffect, useRef, useState, RefObject } from 'react';
 import { Manager, Reference } from 'react-popper';
-
 import Line from '../Line/Line';
-import InfoBox from '../InfoBox/InfoBox';
-
+import { Config, dimensionType, getConfig } from './helpers/config';
+import { getCurrentComponents, ComponentsConfig } from './helpers/components';
 import styles from './RedLiner.module.scss';
 
-type dimensionType = 'height' | 'info' | 'width';
-
-interface Config {
-  color?: string;
-  displayOpts?: dimensionType[] | 'all';
-  infoOpts?: string[];
-}
```

Comment:
The top of this file was getting cluttered so these deletions are moved into a `./helpers/config` file. A `./helpers/components` file was also created for the custom components related stuff.